.github/workflows/s3-cleanup.yml

```diff
@@ -0,0 +1,21 @@
+name: S3-cleanup
+
+on: delete
+
+env:
+  branch-name: ${GITHUB_REF#refs/heads/}
+  bucket-name: 'dokka-snapshots'
+
+jobs:
+  coroutines:
```

Comment:
why are we not cleaning other libraries?

Reply:
Nice catch, it's just misnamed